src/Workspaces/Remote/ServiceHub/Services/CodeAnalysisService_DependentTypeFinder.cs

```diff
@@ -0,0 +1,84 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
```

Comment:
>System [](start = 6, length = 6)

#nullable enable

Reply:
will add in https://github.com/dotnet/roslyn/pull/43804